sourcegraph/jetbrains
# Add subscription tab

It fixes [PLG: Add "Upgrade to Cody Pro" and "Current Usage" settings buttons#149]
(https://github.com/sourcegraph/jetbrains/issues/149)
We need to merge https://github.com/sourcegraph/cody/pull/2274 first
## Test plan
* Log in as Cody Free user
* Check if subscription tab available
* Check if you see correct tier
* Check if you see `Upgrade` and `Check Usage` buttons that are redirecting to `https://sourcegraph.com/cody/subscription` and `https://sourcegraph.com/cody/manage`
* Log in as Cody Pro user
* Check if subscription tab available
* Check if you see correct tier
* Check if you see only `Check Usage` buttons that are redirecting to `https://sourcegraph.com/cody/subscription` and `https://sourcegraph.com/cody/manage`


https://github.com/sourcegraph/jetbrains/assets/9321940/79ccf53d-0685-4b41-815e-511a0beb5f22
